doc/source/reference/maskedarray.generic.rst

```diff
@@ -391,7 +391,7 @@ required to avoid propagation of any modification of the mask to the original.
    >>> mx[1] = -1
    >>> mx
    masked_array(data = [1 -1 3],
-                mask = [False  True False],
+                mask = [False  False False],
```

Comment:
One of these spaces need to be removed, if we ever want this to pass doctests in future